pitch.md

```diff
@@ -1,20 +1,86 @@
-# Project Title (replace me)
+# Succinct Scheduling
 
 ## Describe your project in five words
+Describe schedules sucinctly and precisely.
 
 ## Audience
 _Who are you helping, with your language? What is that person's
 experience like now? How would their experience be better if you could help 
 them?_
 
+This project is designed to serve individuals who need to schedule meetings
+among a group of people.  The language would provide a means by which an
+individual can specify their preference related to scheduling.  With more than
+a handful of reasonably busy individuals it is unlikely that there will be a
+meeting time which is the top choice of everyone involved.
+
+Currently the de-facto standards for scheduling are one
+of the following two methods:
+ * Everyone send out their availability for a small set of times and a leader
+picks the times that seem to work best for a plurality.
+ * Everyone uses some software, such as a WhenIsGood&#8482; or When2Meet&#8482;
+and choses a binary yes/no for each time slot.
+
+Beyond the issue of there not being a single time that works for everyone,
+there is also an issue in which individuals will be conservative in listing
+times that they could theoretically attend, as Daniel Sonner pointed out,
+some people will paint over the times that are *best* for them, but not
+*all* the times which could work for them.
+
+Both of these issues (inability to find a time preferred by all individuals
+and people being extra conservative with respect to their availability) are
+addressed by allowing for greater expressiveness in describing one's schedule.
+This is because allowing for a gradation of availability (from totally free, to
+somewhat available, to completely busy) might make people less likely to
+game the system by saying that they are completely at times that are just
+suboptimal for them.  It also means that there is more information which a
+clever algorithm could make of when deciding which of two conflicts is worse.
```

Comment:
I know I mentioned this during our offline discussion, but will mention it here for you records:

How will "deciding which of two conflicts is worse" be determined? Will the handling of availability conflicts and potentially differing degrees of "badness" be specified by the user or coded directly into your solver?

Personally, I think to begin with, it may be best to leave it coded directly into your algorithm but with sufficient abstraction you could pull that component out and make it configurable by a user through a DSL.